source/htslib/bgzf.d

```diff
@@ -34,6 +34,7 @@ import core.sys.posix.sys.types;
 
 import htslib.hfile : hFILE;
 import htslib.kstring;
+import htslib.thread_pool;
```

Comment:
Is it actually used in this file?

Reply:
Needed for the opaque `hts_tpool` struct declaration. Before I would have to cast a `hts_tpool` from `htslib.bgzf` to the `hts_tpool` type in `htslib.thread_pool`, if I needed to use the same thread pool for both `htslib.bgzf` and other `htslib` functions.

Reply:
I should probably limit the import to only `hts_tpool`.